README.md

```diff
@@ -10,8 +14,7 @@ Created by [@jamiepine]() | Sponsored and maintained by the [Notify Team]()
 
 <p align="left">
   <a href="https://discord.gg/RjG8ShB"><img src="https://discordapp.com/api/guilds/658189217746255881/embed.png" alt="Join Discord"></a>
-  <a href="https://patreon.com/jamiepine"><img src="https://img.shields.io/badge/donate-patreon-F96854.svg" alt="Donate on patreon"></a>
-  <a href="https://twitter.com/jamiepine"><img src="https://img.shields.io/twitter/follow/jamiepine.svg?label=Jamie's Twitter" alt="Follow on twitter"></a>
+   <a href="https://twitter.com/jamiepine"><img src="https://img.shields.io/twitter/follow/jamiepine.svg?label=Jamie's Twitter" alt="Follow on twitter"></a>
   <a href="https://twitter.com/pulseframework"><img src="https://img.shields.io/twitter/follow/pulseframework.svg?label=Pulse+Twitter" alt="Follow Pulse on twitter"></a>
 </p>
```

Comment:
I pushed some updates to the readme and docs today, but this looks out of date to me. I suspect git hasn't fully updated this to reflect those somehow, but this section was entirely changed. (The changes were already on the `docs` branch I had created for a refresh from a while ago, I just merged them in today)

Could you see what's outdated here make sure the PR can merge properly? I'll look at it again soon if you aren't able to get to it, but it would be massively helpful!